tsconfig.json

```diff
@@ -1,795 +1,508 @@
-      ],
-      "js-tokens": [
-        "./node_modules/js-tokens-BABEL_8_BREAKING-true"
-      ],
-      "regexpu-core": [
-        "./lib/regexpu-core.d.ts"
-      ],
-      "to-fast-properties": [
-        "./node_modules/to-fast-properties-BABEL_8_BREAKING-true"
-      ],
-      "slash": [
-        "./node_modules/slash-BABEL_8_BREAKING-true"
-      ],
-      "kexec": [
-        "./lib/kexec.d.ts"
-      ]
+    "skipLibCheck": false
+  },
+  "include": [
+    "packages/babel-parser/typings/*.d.ts",
+    "dts/**/*.d.ts"
+  ],
+  "references": [
+    {
+      "path": "./codemods/babel-plugin-codemod-object-assign-to-object-spread"
+    },
+    {
+      "path": "./codemods/babel-plugin-codemod-optional-catch-binding"
+    },
+    {
+      "path": "./eslint/babel-eslint-parser"
+    },
+    {
+      "path": "./packages/babel-cli"
+    },
+    {
+      "path": "./packages/babel-helper-annotate-as-pure"
+    },
+    {
+      "path": "./packages/babel-helper-builder-binary-assignment-operator-visitor"
+    },
+    {
+      "path": "./packages/babel-helper-builder-react-jsx"
+    },
+    {
+      "path": "./packages/babel-helper-create-class-features-plugin"
+    },
+    {
+      "path": "./packages/babel-helper-create-regexp-features-plugin"
+    },
+    {
+      "path": "./packages/babel-helper-import-to-platform-api"
+    },
+    {
+      "path": "./packages/babel-helper-member-expression-to-functions"
+    },
+    {
+      "path": "./packages/babel-helper-optimise-call-expression"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
+    },
+    {
+      "path": "./packages/babel-helper-plugin-utils"
```

Comment:
why this path repeated so many times?

Reply:
It doesn't cause problems but yes it's a bug in our script to generate tsconfig files -- thanks for catching it!